test/JavaScriptTests/patchConsoleObjectTests.js

```diff
@@ -0,0 +1,125 @@
+var assert = require('assert');
+var patchConsoleObject = require("../../bin/Debugger.Sample/PatchConsoleObject.js")
+
+var dConsoleDebug = "";
+var dConsoleError = "";
+var dConsoleWarn = "";
+var dConsoleLog = "";
+var dConsoleInfo = "";
+var consoleDebug = "";
+var consoleError = "";
+var consoleWarn = "";
+var consoleLog = "";
+var consoleInfo = "";
+
+var consoleDebugger = {
+    log(msg){
+        dConsoleLog += msg;
```

Comment:
I was thinking there would be one "consoleOutput" string which would have a message with a newline so you can tell which output function was called, e.g. consoleDebugger.log() would do:
```
log(msg) {
   consoleOutput += "DEBUGGER LOG: " + msg + "\n";
}
```
so other messages would start with "DEBUGGING INFO: " or "CONSOLE LOG: " etc.

Each test would reset consoleOutput to an empty string, perform the output, then compare the consoleOuput string to the expected.